helm/scylla-operator/templates/scylladbmonitoring_prometheus_clusterrole_def.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scylladb:aggregate-to-scylladb-monitoring-prometheus
+  labels:
+    rbac.operator.scylladb.com/aggregate-to-scylladb-monitoring-prometheus: "true"
+rules:
+- apiGroups:
+  - ""
+  resources:
+#  - nodes
```

Comment:
lets remove commented out code. If these would be required, e2e would catch it.

Reply:
done